### DISCUSSION TITLE
Crashes

### DISCUSSION THREAD

**Discussion by doubleh87** [martinet101/ElevenClock#619] [Q&A] [answered]
I installed it through Microsoft Store. Upon launching it, it runs for less than a second, and then it crashes without any error message. I can't check what version I have, but I installed it like 10 minutes ago. I tried opening the setting first, but same thing happens. I also tried uninstalling it and installing it again. Could anyone help me troubleshoot this please?
[.elevenclock.zip](https://github.com/martinet101/ElevenClock/files/8084437/default.elevenclock.zip)

**Comment by marticliment** [comment, marked as answer]
Yes, you probably installled version 3.2.1/3.2.2 that wewe crashing. I already had updated the version on ms store to version 3.2.3, but they have approved it only 3 hours ago, so this is what might be causing the issue

**Reply by marticliment** [reply]
Try reinstalling

**Reply by doubleh87** [reply]
did and it's been working well so far. thank you!
